.github/workflows/tests.yml

```diff
@@ -36,9 +36,12 @@ jobs:
       # Workaround for required env vars
       env:
         TOKEN_SECRET_KEY: 1234
+        ICONS_S3_BUCKET: mybucket
+        ICONS_S3_BUCKET_REGION: af-south-1
```

Comment:
think everything should go into eu-west-1?

Reply:
just a stub so it doesn't matter